mod-data-export/mod-data-export.postman_collection.json

```diff
@@ -1826,6 +2183,37 @@
 					"        invalidFileDefinition : {",
 					"            \"fileName\": \"test_file.txt\"",
 					"        }",
+					"    },",
+					"    mappingProfile: {",
+					"      \"id\": \"3798f546-7afb-11ea-bc55-0242ac130003\",",
+					"      \"name\": \"Mapping profile\",",
+					"      \"description\": \"Mapping profile description\",",
+					"      \"recordTypes\": [",
+					"        \"INSTANCE\"",
+					"      ],",
+					"      \"outputFormat\": \"MARC\",",
+					"      \"transformations\": [",
+					"        {",
+					"          \"fieldId\": \"electronicAccess.linkText\",",
+					"          \"path\": \"$.electronicAccess[*].linkText\",",
+					"          \"transformation\": \"001\",",
+					"          \"enabled\": \"true\",",
+					"          \"recordType\": \"INSTANCE\"",
+					"        }",
+					"      ],",
+					"      \"userInfo\": {",
+					"        \"firstName\": \"User\",",
+					"        \"lastName\": \"User\",",
+					"        \"userName\": \"@username\"",
+					"      },",
+					"      \"metadata\": {",
```

Comment:
Metadata would be added by RMB, we need to send it while creating

Reply:
Thank you. I will remove metadata then